Sources/Status.swift

```diff
@@ -62,123 +62,123 @@ public enum Status : Int, CustomStringConvertible {
     case .NonAuthoritiveInformation:
-      return "NON-AUTHORITATIVE INFORMATION"
+      return "Non-Authoritative Information"
     case .NoContent:
-      return "NO CONTENT"
+      return "No Content"
     case .ResetContent:
-      return "RESET CONTENT"
+      return "Reset Content"
     case .PartialContent:
-      return "PARTIAL CONTENT"
+      return "Partial Content"
     case .MultiStatus:
-      return "MULTI STATUS"
+      return "Multi-Status"
     case .AlreadyReported:
-      return "ALREADY REPORTED"
+      return "Already Reported"
     case .IMUsed:
-      return "IM USED"
+      return "IM Used"
     case .MultipleChoices:
-      return "MULTIPLE CHOICES"
+      return "Multiple Choices"
     case .MovedPermanently:
-      return "MOVED PERMANENTLY"
+      return "Moved Permanently"
     case .Found:
-      return "FOUND"
+      return "Found"
     case .SeeOther:
-      return "SEE OTHER"
+      return "See Other"
     case .NotModified:
-      return "NOT MODIFIED"
+      return "Not Modified"
     case .UseProxy:
-      return "USE PROXY"
+      return "Use Proxy"
     case .Reserved:
-      return "RESERVED"
+      return "Reserved"
     case .TemporaryRedirect:
-      return "TEMPORARY REDIRECT"
+      return "Temporary Redirect"
     case .PermanentRedirect:
-      return "PERMANENT REDIRECT"
+      return "Permanent Redirect"
     case .BadRequest:
-      return "BAD REQUEST"
+      return "Bad Request"
     case .Unauthorized:
-      return "UNAUTHORIZED"
+      return "Unauthorized"
     case .PaymentRequired:
-      return "PAYMENT REQUIRED"
+      return "Payment Required"
     case .Forbidden:
-      return "FORBIDDEN"
+      return "Forbidden"
     case .NotFound:
-      return "NOT FOUND"
+      return "Not Found"
     case .MethodNotAllowed:
-      return "METHOD NOT ALLOWED"
+      return "Method Not Allowed"
     case .NotAcceptable:
-      return "NOT ACCEPTABLE"
+      return "Not Acceptable"
     case .ProxyAuthenticationRequired:
-      return "PROXY AUTHENTICATION REQUIRED"
+      return "Proxy Authentication Required"
     case .RequestTimeout:
-      return "REQUEST TIMEOUT"
+      return "Request Timeout"
     case .Conflict:
-      return "CONFLICT"
+      return "Conflict"
     case .Gone:
-      return "GONE"
+      return "Gone"
     case .LengthRequired:
-      return "LENGTH REQUIRED"
+      return "Length Required"
     case .PreconditionFailed:
-      return "PRECONDITION FAILED"
+      return "Precondition Failed"
     case .RequestEntityTooLarge:
-      return "REQUEST ENTITY TOO LARGE"
+      return "Request Entity Too Large"
     case .RequestURITooLong:
-      return "REQUEST-URI TOO LONG"
+      return "Request-URI Too Long"
     case .UnsupportedMediaType:
-      return "UNSUPPORTED MEDIA TYPE"
+      return "Unsupported Media Type"
     case .RequestedRangeNotSatisfiable:
-      return "REQUESTED RANGE NOT SATISFIABLE"
+      return "Requested Range Not Satisfiable"
```

Comment:
This one is a full sentence and could be mostly lower-cased as it is in RFC2616:

```
| "416"  ; Section 10.4.17: Requested range not satisfiable
```